asilib/asi/themis.py

```diff
@@ -59,12 +59,16 @@ def themis(
         If True, will download the data from the internet, regardless of
         wether or not the data exists locally (useful if the data becomes
         corrupted).
-    custom_alt: bool
-        If True, asilib will calculate (lat, lon) skymaps assuming a spherical Earth. Otherwise, it will use the official skymaps (Courtesy of University of Calgary).
+    custom_alt: str, default None
+        When selected, there are two options for skyma's between official sky maps:
+        If 'Geodetic', asilib will calculate (lat, lon) skymaps assuming a spherical Earth. Otherwise, it will use the official skymaps (Courtesy of University of Calgary).
 
         .. note::
         
             The spherical model of Earth's surface is less accurate than the oblate spheroid geometrical representation. Therefore, there will be a small difference between these and the official skymaps.
+
+        If 'Interp', asilib will calculate the (lat,lon) sky maps assuming that the interpolation between official maps is linear. This was supported by personal conversations with Dr. Eric Donovan of the University of Calgary
```

Comment:
Same with changing 'Interp' lowercase. This should be also edited in the REGO and TREx functions.